simple_history/models.py

```diff
@@ -113,7 +113,10 @@ def create_history_model(self, model, inherited):
         """
         Creates a historical model to associate with the model provided.
         """
-        attrs = {'__module__': self.module}
+        attrs = {
+            '__module__': self.module,
+            '_history_excluded_fields': self.excluded_fields
```

Comment:
Still don't love adding this on the model itself. Will think about this about and think about any cleaner solutions. Going to be merging some `bulk_history_create` updates anyway so this will need to be rebased. I'll let you know if I can think of any other way of doing this. Otherwise we'll go with this.

Reply:
For reference, [this](https://code.djangoproject.com/ticket/5793) is why we can't put `excluded_fields` on the meta class. So I'm fine with the current implementation.